### DISCUSSION TITLE
What's the best way to edit Fastfile in Xcode with syntax highlighting?

### DISCUSSION THREAD

**Discussion by bart-kneepkens** [fastlane/fastlane#20554] [Q&A]
I like to use Xcode to edit the Fastfile - however, since the file has no extension, I have to manually click 'Editor' > 'Syntax Highlighting' > 'Ruby' every time. I tried to add a `.rb` extension to the Fastfile, which prevents me from having to do this manual step, but then fastlane can't find the Fastfile anymore: `Could not find fastlane in current directory`.

Does anyone have tips or tricks on how to easily edit the Fastfile using Xcode?

**Comment by jbload** [comment]
Select your Fastfile in the Project Navigator on the left then open the File Inspector (`Option + Command + 1`) on the right and you'll see a Type field. After setting the Type field to "Ruby Script", the next time you open the file you should see Ruby syntax highlighting. I can confirm it works in Xcode 14.2 but it likely works in earlier versions as well.

![Screenshot](https://user-images.githubusercontent.com/1613883/208239784-85b2eb92-18ea-4dcb-b1ca-74dc8131eaba.png)
